main/store/state/types/media.ts

```diff
@@ -1,13 +1,5 @@
 import { z } from 'zod'
+import { v39MediaSchema } from '../../migrate/migrations/39'
 
-export const MediaSchema = z.object({
-  source: z.string(),
-  format: z.union([z.literal('image'), z.literal('video'), z.literal('')]),
-  cdn: z.object({
-    main: z.string().optional(),
-    thumb: z.string().optional(),
-    frozenThumb: z.string().optional()
-  })
-})
-
+export const MediaSchema = v39MediaSchema
```

Comment:
:+1: 

I've been trying to think of a pattern where we can use the latest version like this without having to manually update all the links if a new migration is written. if you can think of any ideas on this front lmk